test/unit/three_scale/redis_config_test.rb

```diff
@@ -43,5 +43,51 @@ class RedisConfigTest < ActiveSupport::TestCase
       assert config.key? :size
       assert_equal 5, config[:size]
     end
+
+    %i[ca_file ca_path].each do |param|
+      test "it takes given #{param} when provided" do
+        value = 'any_value'
+        raw_config = { url: 'rediss://my-secure-redis/1', ssl_params: {}}
+        raw_config[:ssl_params][param] = value
+
+        result = RedisConfig.new(raw_config)
+
+        assert result.key? :ssl_params
+        assert result[:ssl_params].key? param
+        assert_equal value, result[:ssl_params][param]
+      end
+    end
+
+    test 'it trusts on CAs in config/ca_cert.pem if no ca_file or ca_path are provided' do
+      FakeFS.with_fresh do
+        FakeFS::FileSystem.clone(file_fixture_path)
+        FakeFS::FileSystem.clone(Rails.root.join('config'))
+        FileUtils.cp file_fixture('ca_cert.pem'), Rails.root.join('config')
```

Comment:
you can just create an empty file instead of keeping a fixture in the repo
```rb
FileUtils.touch
```

Reply:
Fixed https://github.com/3scale/porta/pull/3572/commits/fcc87b20bd83512cece9f93bb9de3f642a6dcf36